orc8r/cloud/go/syncstore/digest_utils.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Nit: "utils" is an unproductive naming pattern. Just call this digest.go and digest_test.go